### ISSUE TITLE
Overlap Taskbar

### ISSUE THREAD

**Issue by kapilmoondra** [nwjs/nw.js#6227]
NWJS Version : 0.22.3
Operating System : Windows/Mac/Linux

### Expected behavior

Transparent window shall overlap taskbar/dockbar

### Actual behavior

taskbar/dockbar is not overlapped. Instead our window is displayed beneath the taskbar/dockbar

### How to reproduce

1. Create and open a window in Kiosk mode
2. Taskbar/dockbar is displayed on top of window

**Comment by Christywl** [comment]
I can't reproduce this issue on Linux/Windows with nwjs-sdk-v0.26.0. The transparent window in kiosk mode overlaps the taskbar. Right-click in the taskbar, you could see some options of nwjs, not the taskbar options.
![image](https://user-images.githubusercontent.com/20039821/31978341-8c9723c0-b973-11e7-86db-5d084e00de2c.png)

**Comment by Yonezpt** [comment]
@kapilmoondra You are using a relatively old version of NWJS, 0.22.3 when the latest version is 0.26.0 https://nwjs.io/downloads

It's possible the recent version has your issue fixed.

**Comment by kapilmoondra** [comment]
Yes. This issue is now resolved. However if I press CTRL+ALT+TAB and select another window, Task bar appears back on top of kiosk window. If I click on kiosk window, it will again come on top of Taskbar.

**Comment by Christywl** [comment]
Is it a expected behavior? @rogerwang

**Comment by kapilmoondra** [comment]
Any update @rogerwang ?

**Comment by rogerwang** [comment]
CC @jtg-gg the maintainer on the transparency feature.

**Comment by jtg-gg** [comment]
i am a bit busy now, will look into this once i am "free"
